scripts/version

```diff
@@ -21,7 +21,11 @@ SUFFIX="-${ARCH}"
 
 TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
+HELM_TAG=${TAG:-${VERSION}}
+HELM_VERSION=${HELM_TAG/v/}
 
 if echo $TAG | grep -q dirty; then
     TAG=dev
+    HELM_TAG=dev
+    HELM_VERSION=0.0.0-dev
```

Comment:
Why do we need them? Where are they used?

Reply:
We need them in scripts/package-helm. They were not present in the original PR that added scripts/package-helm